src/components/ha-selector/ha-selector-select.ts

```diff
@@ -80,16 +119,39 @@ export class HaSelectSelector extends LitElement {
     });
   }
 
-  static get styles(): CSSResultGroup {
-    return css`
-      ha-select {
-        width: 100%;
-      }
-      mwc-formfield {
-        display: block;
-      }
-    `;
+  private _removeItem(ev): void {
+    (this.value as string[])!.splice(ev.target.idx, 1);
+
+    fireEvent(this, "value-changed", {
+      value: this.value,
+    });
+    this.requestUpdate();
   }
+
+  private _comboBoxValueChanged(ev: CustomEvent): void {
+    ev.stopPropagation();
+    const newValue = ev.detail.value;
+
+    if (this.disabled || newValue === "") {
```

Comment:
- You need to validate that the new value is an allowed value
- You need to empty the combobox after picking an option

Reply:
Actually, should it be a combobox or just a dropdown ?

Reply:
Combo box allows for select and custom values

Reply:
If custom value is not set then it will only allow allowed values

I tried to empty the combobox but couldn't figure it out haha